lib/kernel/vfio_container.cpp

```diff
@@ -41,8 +42,7 @@ using namespace villas::kernel::vfio;
   #define VFIO_NOIOMMU_IOMMU 8
 #endif
 
-static
-std::array<std::string, EXTENSION_SIZE> construct_vfio_extension_str() {
+static std::array<std::string, EXTENSION_SIZE> construct_vfio_extension_str() {
```

Comment:
```suggestion
static
std::array<std::string, EXTENSION_SIZE> construct_vfio_extension_str() {
```

Please keep the coding style :)

Reply:
We should revisit #101